library/ssl_misc.h

```diff
@@ -1258,6 +1265,57 @@ static inline psa_key_id_t mbedtls_ssl_get_opaque_psk(
 
     return( MBEDTLS_SVC_KEY_ID_INIT );
 }
+
```

Comment:
As this point we're still in a `#if defined(MBEDTLS_KEY_EXCHANGE_SOME_PSK_ENABLED)` block (ending at line 1321), which shouldn't apply to the next two definitions.

Reply:
Moved out of `MBEDTLS_KEY_EXCHANGE_SOME_PSK_ENABLED`.